File: VB/DiagramFromTreeVB/Models/Department.vb
Public Class Department
    Public Property ID As Integer
    Public Property ParentID As Integer?
    Public Property DepartmentName As String

    Public Sub New(ByVal id As Integer, ByVal parentId As Integer?, ByVal departmentName As String)
        Me.ID = id
        Me.ParentID = parentId
        Me.DepartmentName = departmentName
    End Sub

    Public Sub New()
    End Sub
End Class

File: CS/DiagramFromTree/Models/Department.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DiagramFromTree.Models
{
    public class Department
    {
        public int ID { get; set; }
        public int? ParentID { get; set; }
        public string DepartmentName { get; set; }

        public Department(int id, int? parentId, string departmentName)
        {
            ID = id;
            ParentID = parentId;
            DepartmentName = departmentName;
        }

        public Department() { }
    }
}